common/src/test/java/org/opensearch/ml/common/transport/agent/MLAgentDeleteRequestTest.java

```diff
@@ -56,10 +59,46 @@ public void validate_Failure() {
     }
 
     @Test
-    public void fromActionRequest() throws IOException {
+    public void fromActionRequest_Success() throws IOException {
```

Comment:
In java we use CamelCasing, not snake_casing

Reply:
I checked the testmethod in other packages under ml-common, most of the test Success case are using camel case, that's what I looked up when I write it.

https://github.com/opensearch-project/ml-commons/blob/06f8b2aab36b91386792a284aeaeda9e8c026781/common/src/test/java/org/opensearch/ml/common/transport/task/MLTaskGetResponseTest.java#L48

https://github.com/opensearch-project/ml-commons/blob/06f8b2aab36b91386792a284aeaeda9e8c026781/plugin/src/test/java/org/opensearch/ml/task/MLExecuteTaskRunnerTests.java#L150

https://github.com/opensearch-project/ml-commons/blob/06f8b2aab36b91386792a284aeaeda9e8c026781/memory/src/test/java/org/opensearch/ml/memory/action/conversation/UpdateConversationRequestTests.java#L91

Reply:
I see, ok approving then.